### PR TITLE
fix: disable 'create wallet' link when unlocking wallet

### PR DESCRIPTION
**Pull request by theborakompanioni**
Disable "Create Wallet" button while unlocking a wallet is in progress.
Based on and blocked by https://github.com/joinmarket-webui/joinmarket-webui/pull/351.

### CODE REVIEW

File: src/components/Wallets.jsx [hunk 1]
```diff
@@ -54,16 +56,20 @@ export default function Wallets({ currentWallet, startWallet, stopWallet }) {
...
 
+        setIsUnlocking(false)
+
         const { walletname: unlockedWalletName, token } = body
         startWallet(unlockedWalletName, token)
         navigate(routes.wallet)
       } catch (e) {
         const message = e.message.replace('Wallet', walletName)
         setAlert({ variant: 'danger', dismissible: false, message })
+        setIsUnlocking(false)
```
Comment (dnlggr) [inline src/components/Wallets.jsx:72]: Maybe put it in a `finally` clause?

Comment (theborakompanioni) [reply src/components/Wallets.jsx:72]: I think this is not possible due to `navigate` unmounting the component. Will try and report back.

Comment (theborakompanioni) [reply src/components/Wallets.jsx:72]: Funny how I get 
```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a `useEffect` cleanup function.
```
in Chrome, but not in Firefox. It seems I have not fully grasp the component lifecycle as of yet. Would have thought it is the same in both browsers. Is it okay to leave it unchanged?

Comment (dnlggr) [reply src/components/Wallets.jsx:72]: > I think this is not possible due to navigate unmounting the component. 

Makes sense! ✔️

**Review by dnlggr — APPROVED** [review]

**Pull request merged**